concordium-node/src/connection/low_level.rs

```diff
@@ -307,7 +307,7 @@ impl ConnectionLowLevel {
         recv_xx_msg!(self, len, "C");
         let payload = self.socket_buffer.slice(len)[DHLEN + MAC_LENGTH..]
             [..len - DHLEN - MAC_LENGTH * 2]
-            .try_into()?;
+            .to_vec();
```

Comment:
I suppose the same could be done for `process_msg_a` and `process_msg_b` ?

Reply:
Changed also in those methods.